cassandra-analytics-core/src/main/java/org/apache/cassandra/bridge/CassandraBridgeFactory.java

```diff
@@ -124,6 +124,12 @@ private static String bridgeResourceName(@NotNull String label)
         return "/bridges/" + label + "-bridge.jar";
     }
 
+    @NotNull
+    private static String typesResourceName(@NotNull String label)
+    {
+        return "/bridges/" + label + "-types.jar";
+    }
```

Comment:
nit: this method and the 2 methods above can be merged, e.g. 
```java
    private static String jarResourceName(String[] parts)
    {
        return "/bridges/" + String.join("-", parts) + ".jar";
    }

// usage
jarResourceName(label);
jarResourceName(label, "types);
```

Please feel free to ignore this nit.